File: Python/1 - Iniciante/1133.py
# coding: UTF - 8

a = int(input())
b = int(input())

if a > b:
    maior = a
    menor = b + 1
else:
    maior = b
    menor = a + 1

while menor < maior:
    if menor % 5 == 2 or menor % 5 == 3:
        print(menor)
    menor += 1

File: go/1133.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	var x int
	var y int
	menor := 0
	maior := 0

	fmt.Scan(&x)
	fmt.Scan(&y)

	if x > y {
		maior = x
		menor = y
	} else {
		maior = y
		menor = x
	}

	for i := menor + 1; i < maior; i++ {
		if i%5 == 2 || i%5 == 3 {
			fmt.Println(i)
		}
	}
}

// Código feito por Gabriel Gomes 2022
